builder/bundles.go

```diff
@@ -436,17 +436,43 @@ func genUpdateBundleSpecialFiles(chrootDir string, b *Builder) error {
 	return ioutil.WriteFile(filepath.Join(swupdDir, "format"), []byte(b.State.Mix.Format), 0644)
 }
 
+func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int) error {
+	var err error
+
+	args := merge(packagerCmd, "--installroot="+baseDir, "install", "--downloadonly")
+	args = append(args, rpmList...)
+
+	for retries := 0; retries < maxRetries; retries++ {
+		_, err = helpers.RunCommandOutputEnv(args[0], args[1:], []string{"LC_ALL=en_US.UTF-8"})
+		if err == nil {
+			return err
```

Comment:
Doesn't really matter here, but you could also call `break` to end the `for` loop. No need to change it though.